### DISCUSSION TITLE
Create additional Workspace for same OIDC Provider

### DISCUSSION THREAD

**Discussion by mirisbowring** [outline/outline#6389] [Self Hosting] [answered]
Hi,

I just setup an outline instance on my homeserver.
I am using authentik as OIDC and all my users have an account there.

I don't want to end up all users in the same workspace.
I've read, that multiple workspaces for the same IDP are possible:
https://www.getoutline.com/changelog/multiple-workspaces

How can i configure it?
Can i name a specific workspace in an OIDC scope?

**Comment by tommoor** [comment, marked as answer]
Multiple workspaces is currently a cloud-only feature
